### ISSUE TITLE
helm list should use consistent separators between ouput columns, or provide a parsable output option

### ISSUE THREAD

**Issue by wknapik** [kubernetes/helm#3389]
Unlike, say, kubectl, helm does not provide a switch to produce easily parsable (json, yaml, whatever) output, so anyone automating the use of helm is forced to scrape the output that was meant for human consumption.

So for now we need to parse the output of, for example, `helm list` and "cut out" what we need. This is made harder by having inconsistent separators between columns.

Here's an illustration of the problem:
```
# helm list|cut -f1|sed 's/$/X/g'
NAME       X
foo        X
foo-ba     X
foo-bar    X
foo-barbazzX
foo-bazquuxX
bar        X
# helm list|cut -d" " -f1|sed 's/$/X/g'
NAMEX
fooX
foo-baX
foo-barX
foo-barbazz     2X
foo-bazquux     3X
barX
#
```

First we cut using "\t" as delimeter, then using " ". Both work sometimes and fail other times, neither is valid all the time.

And sure, you can do something like `grep -oP '^\S+'`, or `sed -r 's/^(\S+).*/\1/g'`, or some such, but those are just workarounds for the use of inconsistent delimeters, which is bad whenever the output is likely to be scraped automatically.

And before someone says "just use `helm list -q`" - no, that's not an answer either. If I need to do some matching on full output, before I extract the release name, `-q` is not usable.

So to sum up - we really should be able to get output in an automatically parsable format, but until that feature is implemented, scraping the only format available should not be made more difficult by mixing tabs and spaces as column separators.

**Comment by bacongobbler** [comment]
closing as a duplicate of #1534 which there's a PR available at #2950.
